subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -2832,6 +2847,132 @@ void ull_conn_default_tx_time_set(uint16_t tx_time)
 }
 #endif /* CONFIG_BT_CTLR_DATA_LENGTH */
 
+uint8_t ull_fsu_update_eff(struct ll_conn *conn)
+{
+	uint8_t fsu_changed = 0U;
```

Comment:
Given that we now are using C17 in Zephyr and that `bool` is an actual type, should we consider using `bool` over `uint8_t` for boolean values in the controller?